dynamiq/nodes/rankers/cohere.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, ClassVar, Literal
+
+from litellm import rerank
```

Comment:
Please add `rerank` as private attr on init similar to completion and other litellm helpers. Please optimize import in a similar way to speed up loading.